orders/views.py

```diff
@@ -10,43 +10,85 @@
 
 class AddItemView(View):
     @check_authorization
-    def get(self, request, pk, name):
+    def get(self, request, pk, color):
         try:
             item            = Item.objects.get(id=pk)
-            item_color      = Color.objects.get(name=name)
+            item_color      = Color.objects.get(name=color)
             order           = Order.objects.get(Q(customer = request.user.id)& Q(order_status = UNSETTLED))
             check_cart      = OrderItem.objects.filter(Q(order=order.id)& Q(item=item)& Q(color=item_color)).exists()
             if check_cart:
-                cart = OrderItem.objects.get(Q(order=order.id)& Q(item=item)& Q(color=item_color))
+                cart         = OrderItem.objects.get(Q(order=order.id)& Q(item=item)& Q(color=item_color))
                 cart.amount += 1
                 cart.save()
-            else:
-                cart = OrderItem.objects.create(order = order, 
-                                            item = item,
-                                            amount = 1,
-                                            color=item_color)
+                return HttpResponse(status = 200)
+            cart = OrderItem.objects.create(order   = order, 
+                                            item    = item,
+                                            amount  = 1,
+                                            color   = item_color)
```

Comment:
줄바꿈 후에는 가장 왼쪽부터 4칸만 띄워서 정리 부탁 드립니다.
```suggestion
            cart = OrderItem.objects.create(
                order   = order, 
                item    = item,
                amount  = 1,
                color   = item_color)
```